src/test/features/response/form/models/timeline.ts

```diff
@@ -118,11 +118,11 @@ describe('Timeline', () => {
 
       expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
 
-      for (let i = 0; i < MAX_NUMBER_OF_EVENTS + 1; i++) {
+      for (let i = 0; i < MAX_NUMBER_OF_ROWS + 1; i++) {
         actual.appendRow()
       }
 
-      expect(actual.rows.length).to.be.eq(MAX_NUMBER_OF_EVENTS)
+      expect(actual.rows.length).to.be.eq(MAX_NUMBER_OF_ROWS)
```

Comment:
Same here - test checks for something different (MAX_NUMBER_OF_ROWS) than it says (20). In this case, it seems that the inconsistency was already there, so it's up to you to decide it you want to fix it.

Reply:
done